src/ds/ds-private.h

```diff
@@ -123,6 +123,7 @@ namespace librealsense
             RECPARAMSGET = 0x7E,     // Retrieve depth calibration table in new format (fw >= 5.11.12.100)
             LASERONCONST = 0x7F,     // Enable Laser On constantly (GS SKU Only)
             AUTO_CALIB = 0x80,      // auto calibration commands
+            HKR_THERMAL_COMPENSAITON = 0x84, // Control HKR thermal compensation
```

Comment:
Not sure 'HKR' should appear here...

Reply:
It also appears for other opcodes, e.g. GET_HKR_CONFIG_TABLE

Reply:
Also, there's a typo in there

Reply:
Typo fixed.
Keeping it here for now, will consider moving all HKR specific opcodes to d500-private on a separate PR.